pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -709,7 +709,7 @@ benchmarks! {
 			state.requests().get(&collator),
 			Some(&DelegationRequest {
 				collator,
-				amount: 5u32.into(),
+				amount: <<T as Config>::MinDelegatorStk as Get<BalanceOf<T>>>::get(),
```

Comment:
```suggestion
				amount: bond_less,
```
We can use the parameter which is set equal to this in the `verify` block as well.

Reply:
Oh, I think you're right :+1: